website/docs/d/workspace.html.markdown

```diff
@@ -60,7 +60,7 @@ In addition to all arguments above, the following attributes are exported:
 * `trigger_patterns` - List of [glob patterns](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/settings/vcs#glob-patterns-for-automatic-run-triggering) that describe the files Terraform Cloud monitors for changes. Trigger patterns are always appended to the root directory of the repository.
 * `vcs_repo` - Settings for the workspace's VCS repository.
 * `working_directory` - A relative path that Terraform will execute within.
-* `execution_mode` - Indicates the [execution mode](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/settings#execution-mode) of the workspace. Possible values include `remote`, `local`, or `agent`.
+* `execution_mode` - **Deprecated** Indicates the [execution mode](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/settings#execution-mode) of the workspace. Use the `tfe_workspace_settings` resource instead.
```

Comment:
Hmm, this probably isn't called for in the data-source, since that's read-only... right?

Reply:
I did this because it's not the full picture any longer because `execution_mode` can be set from the organization default or from the workspace (as an override) `tfe_workspace_settings` gives access to both pieces of information and contains only optional attributes.

Reply:
Ah ok, thx for that context — I think I still disagree slightly, but I'll make a case in a docs PR.